.github/workflows/test.yml

```diff
@@ -1,33 +0,0 @@
-name: test
```

Comment:
Can we keep the same file name as is bring used in the other repos for consistency? "Rust CI" is not that descriptive as the whole repo is Rust and fact that the file is in workflows already implies it will run as part of the CI